app/src/main/java/ru/yandexpraktikum/marketplace/ui/screens/MainScreen.kt

```diff
@@ -93,8 +97,10 @@ fun MainScreen(onProductClick: (Int) -> Unit) {
                 },
                 placeholder = {
                     Text(
-                        text = stringResource(R.string.search_products),
-                        color = Color(0xFFAAAAAA)
+                        modifier = Modifier.semantics {
+                            contentDescription = searchBarDescription
```

Comment:
Тут можно заменить на вызов функции 
```suggestion
                            contentDescription = stringResource(R.string.searchbar_description)
```
Таким образом, можно от переменной избавиться